<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0.005" tests="3" failures="1">
  <testsuite name="Root Suite" timestamp="2020-01-13T21:18:47" tests="0" failures="0" time="0">
  </testsuite>
  <testsuite name="fibonacci" timestamp="2020-01-13T21:18:47" tests="1" file="/home/samuka/codenation/nodejs-0/test/submit.test.js" failures="1" time="0.005">
    <testcase name="fibonacci should get fibonacci sequence until it passes 350" time="0.005" classname="should get fibonacci sequence until it passes 350">
      <failure message="1: expected [ Array(14) ] to include 0" type="AssertionError"><![CDATA[AssertionError: 1: expected [ Array(14) ] to include 0
    at Context.it (test/submit.test.js:9:27)]]></failure>
    </testcase>
  </testsuite>
  <testsuite name="isFibonacci" timestamp="2020-01-13T21:18:47" tests="2" file="/home/samuka/codenation/nodejs-0/test/submit.test.js" failures="0" time="0">
    <testcase name="isFibonacci should return true when the number is a fibonacci" time="0" classname="should return true when the number is a fibonacci">
    </testcase>
    <testcase name="isFibonacci should return false when the number isn&apos;t a fibonacci" time="0" classname="should return false when the number isn&apos;t a fibonacci">
    </testcase>
  </testsuite>
</testsuites>